frontends/web/src/routes/account/summary/accountssummary.tsx

```diff
@@ -217,7 +216,7 @@ export function AccountsSummary({ accounts, devices }: TProps) {
                   totalBalancePerCoin={ balancePerCoin ? balancePerCoin[keystore.rootFingerprint] : undefined}
                   totalBalance={ accountsTotalBalance ? accountsTotalBalance[keystore.rootFingerprint] : undefined}
                   balances={balances}
-                  lightningNodeState={(lightningConfig.accounts.length && lightningConfig.accounts[0].rootFingerprint === keystore.rootFingerprint) ? nodeState : undefined}
+                  lightningBalance={(lightningConfig.accounts.length && lightningConfig.accounts[0].rootFingerprint === keystore.rootFingerprint) ? lightningBalance : undefined}
```

Comment:
This component doesn't really need `lightningConfig` and `lightningBalance` as those are only used in SummaryBalance component, wdyt about moving as much s possible into SummaryBalance?

you'll probably have to pass `keystore.rootFingerprint` as well somehow for the condition.

Reply:
yeah, I moved the LN logic inside the component and refactored a bit to only pass it the keystore struct, instead of various fields belonging to it.